app/assets/javascripts/dashboard.js

```diff
@@ -54,6 +61,16 @@ MinionPoller = {
     });
   },
 
+  enable_kubeconfig: function(enable) {
+    if (enable) {
+      $("#download-kubeconfig").unbind('click')
+      $("#download-kubeconfig").removeAttr("disabled");
+    } else {
+      $("#download-kubeconfig").bind('click', function(e) { e.preventDefault(); });
+      $("#download-kubeconfig").attr("disabled", true);
+    }
+  },
```

Comment:
> Use .prop('disabled', !enabled) instead of attr and removeAttr

Weird, it didn't work for me, that's why I did it 😕 

> Is the bind and unbind really necessary?

Even if disabled, I could still click it and then I got the download dialog... So it looks like it...